### DISCUSSION TITLE
How to flag a request failed (or deny after approval)?

### DISCUSSION THREAD

**Discussion by kimfr** [sct/overseerr#3314] [Q&A]
Hi there

I'm using overseerr together with sonarr and radarr - some of my external users have requested a movie for example, I approve the request and it's automatically sent to radarr. For some reason, the download never finishes - either because radarr can't find any source or because the download never completes.

I haven't found a way to Deny the requested (or flag it failed) - you can only delete the request which doesn't seem to send a message back to the requester.
I've also tried flagging the download failed within radarr (to the extend it possible) - also doesn't seem to inform the requested or get "picked" up by overseerr.

How do I now handle this situation?

Thanks
Kim

**Comment by TheCatLady** [comment]
A request is only considered failed if it failed to be added to the *arrs. It is not possible to deny a request after approval, as it has already been added to the *arrs for processing.

In this case, it was not a failure. The *arrs will continue to monitor the media and the user will be notified when their request is eventually fulfilled.

**Reply by kimfr** [reply]
The problem however is that the *arrs never completes the download - mainly because the requested item isn't available anymore (happens mainly with media requests for older content).

Perhaps, an enhancement to be able to flag something failed/unavailable within Overseerr which when would
a) Sent a message to requested, possibly with a added comment  
b) delete/remove the item from *arrs (should probably be a configurable option).
